### PR TITLE
Add creator attribute for data points

### PR DESCRIPTION
**Pull request by em-henken**
There is still a bug, which I will need to look into. One with stronger knowledge of the codebase/JavaScript would probably be able to help. Only saves creator if first button is clicked on the URI input screen. However, if you wait to enter it after the first button but before "Confirm URI" button, it saves as an empty string. How to get it saved just based on the Confirm URI button, the last button clicked before going to the next window.

### CODE REVIEW

File: public/components/layout.js [hunk 1]
```diff
@@ -135,9 +146,10 @@ class UserResource extends HTMLElement {
                     + " will note be able to gather additional information about this targeted resource."
                     + " You can supply a different URI or continue with this one.")
                 uriPreview.innerHTML = `<pre>{Not Resolvable}</pre>`
-                localStorage.setItem("userResource", JSON.stringify({"@id":target})) 
+                localStorage.setItem("userResource", JSON.stringify({"@id":target, "creator":objCreator.value}))
```
Comment (tobradovic) [inline public/components/layout.js:149]: So this is adding a new `creator` JSON object within the `userResource` JSON object, right?

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Add the "Name and Email" in <label> tag and keep both label and input tag inside div tag

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
The creator JSON should not be added to JSON when there is no value provided

File: public/components/layout.js [hunk 2]
```diff
@@ -135,7 +145,7 @@ class UserResource extends HTMLElement {
...
```
Comment (thehabes) [inline public/components/layout.js:148]: Do not add the `creator` key if the value is an empty string, `undefined`, or `null`

File: public/components/layout.js [hunk 3]
```diff
@@ -305,9 +317,10 @@ class GeolocatorPreview extends HTMLElement {
                 wrapper = {
                     "@context": ["http://www.w3.org/ns/anno.jsonld", "https://geojson.org/geojson-ld/geojson-context.jsonld"],
                     "type": "Annotation",
+                    "creator": userObj["creator"],
```
Comment (thehabes) [inline public/components/layout.js:320]: Make sure not to add the `creato`r key if the value is an empty string, undefined, or null

**Review by thehabes — CHANGES_REQUESTED** [review]

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Name and URI should be in same div it should not be in different div's

**Comment by thehabes** [comment]
I agree, we want it to look more like

image

Also, that message is a bit technically direct.  I love it, but I am not your typical user.  How about something along the lines of
```
Supply an existing Web Resource URI.  You also have the option to provide a name or E-mail to serve as the creator of this resource.  Use your own name or E-mail to take ownership of the data created!
```

`Use This URI` as the button text should probably also change, since now there is additional input.  Something more generic like  `Save and Next Step` or whatever you come up with.

**Review by thehabes — APPROVED** [review]
There it is!  Looks good, I think we will continually think about how to do the UI.  Still, it feels normal to me to provide my name and resource at the same time, and the button queues tie it all together.

**Review by yashaswini-slu — APPROVED** [review]
All changes are implemented.

**Pull request merged**